AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -36,4 +36,8 @@ public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
                          credentialsProvider: AWSCredentialsProvider) {
         super.init(region: regionType, credentialsProvider: credentialsProvider)
     }
+
+    public init(region regionType: AWSRegionType) {
+        super.init(region: regionType, credentialsProvider: nil)
+    }
```

Comment:
Do we need this? We already have an initializer that takes a `nil`, as used on L32 above.

Reply:
That initializer is overriden by the initilizer in line L35 that take a nonnull `credentialsProvider`. So I cannot init this class outside like this: `AmplifyAWSServiceConfiguration(region: region, credentialsProvider: nil)`